pkg/scannerv4/mappers/mappers.go

```diff
@@ -984,3 +1093,32 @@ func vulnsEqual(a, b *claircore.Vulnerability) bool {
 		a.NormalizedSeverity == b.NormalizedSeverity &&
 		a.FixedInVersion == b.FixedInVersion
 }
+
+// dedupeAdvisories deduplicates repeat advisories out of vulnIDs and returns the result.
+// This function will only filter if ROX_SCANNER_V4_RED_HAT_CSAF is enabled; otherwise,
+// it'll just return the original slice of vulnIDs.
+// This function does not guarantee order is preserved.
+func dedupeAdvisories(vulnIDs []string, protoVulns map[string]*v4.VulnerabilityReport_Vulnerability) []string {
```

Comment:
I was expecting this function to be unnecessary because `vulnsDedup` should do the trick if we vulnerabilities are equivalent after they are "converted" into RHSA:

```
return a.Name == b.Name &&
		a.Description == b.Description &&
		a.Issued == b.Issued &&
		a.Links == b.Links &&
		a.Severity == b.Severity &&
		a.NormalizedSeverity == b.NormalizedSeverity &&
		a.FixedInVersion == b.FixedInVersion
		```

Reply:
Going to keep things as-is. Resolving